README.md

```diff
@@ -21,7 +21,7 @@ cfn-lint, Guard and more:
 
 ## Key features
 
-* **Interactive deployments**: With `rain deploy`, rain packages your CloudFormation templates using [`aws cloudformation package`](https://awscli.amazonaws.com/v2/documentation/api/latest/reference/cloudformation/package.html), prompts you for any parameters that have not yet been defined, shows you a summary of the changes that will be made, and then displays real-time updates as your stack is being deployed. Once finished, you get a summary of the outcome along with any error messages collected along the way - including errors messages for stacks that have been rolled back and no longer exist.
+* **Interactive deployments**: With `rain deploy`, rain packages your CloudFormation templates, prompts you for any parameters that have not yet been defined, shows you a summary of the changes that will be made, and then displays real-time updates as your stack is being deployed. Once finished, you get a summary of the outcome along with any error messages collected along the way - including errors messages for stacks that have been rolled back and no longer exist.
```

Comment:
Did you edit this file directly? It is generated.

Reply:
I didn't edit this file directly.
I tried again in the latest commit, but I got the same result.

```sh
❯ git log -n 1
commit 14666afa1ae8a679862853f4a6c28e01d2b3f99d (HEAD -> main, origin/main, origin/HEAD)
Merge: 7ddc5c2 130e805
Author: Eric Z. Beard <ericzbeard@gmail.com>
Date:   Tue Feb 13 10:59:57 2024 -0800

    Merge pull request #275 from aws-cloudformation/dependabot/go_modules/golang.org/x/term-0.17.0
    
    Bump golang.org/x/term from 0.16.0 to 0.17.0

❯ git status 
On branch main
Your branch is up to date with 'origin/main'.

nothing to commit, working tree clean

❯ go generate ./...
go: downloading golang.org/x/term v0.17.0
go: downloading golang.org/x/sys v0.17.0

❯ git diff README.md 
diff --git a/README.md b/README.md
index 501f01b..f5d283f 100644
--- a/README.md
+++ b/README.md
@@ -21,7 +21,7 @@ cfn-lint, Guard and more:
 
 ## Key features
 
-* **Interactive deployments**: With `rain deploy`, rain packages your CloudFormation templates using [`aws cloudformation package`](https://awscli.amazonaws.com/v2/documentation/api/latest/reference/cloudformation/package.html), prompts you for any parameters that have not yet been defined, shows you a summary of the changes that will be made, and then displays real-time updates as your stack is being deployed. Once finished, you get a summary of the outcome along with any error messages collected along the way - including errors messages for stacks that have been rolled back and no longer exist.
+* **Interactive deployments**: With `rain deploy`, rain packages your CloudFormation templates, prompts you for any parameters that have not yet been defined, shows you a summary of the changes that will be made, and then displays real-time updates as your stack is being deployed. Once finished, you get a summary of the outcome along with any error messages collected along the way - including errors messages for stacks that have been rolled back and no longer exist.
 
 * **Consistent formatting of CloudFormation templates**: Using `rain fmt`, you can format your CloudFormation templates to a consistent standard or reformat a template from JSON to YAML (or YAML to JSON if you prefer). Rain preserves your comments when using YAML and switches use of [intrinsic functions](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference.html) to use the short syntax where possible.
 
@@ -51,8 +51,7 @@ Usage:
 
 Stack commands:
   cat         Get the CloudFormation template from a running stack
-  ccdeploy    Deploy a local template directly using the Cloud Control API (Experimental!)
-  ccrm        Delete a deployment created by ccdeploy (Experimental!)
+  cc          Interact with templates using Cloud Control API instead of CloudFormation
   deploy      Deploy a CloudFormation stack from a local template
   logs        Show the event log for the named stack
   ls          List running CloudFormation stacks
```

Reply:
I guess the sentence is added directly.
> using [`aws cloudformation package`](https://awscli.amazonaws.com/v2/documentation/api/latest/reference/cloudformation/package.html)

Because the sentence doesn't appear in README.tmpl
>* **Interactive deployments**: With `rain deploy`, rain packages your CloudFormation templates, prompts you for any parameters that have not yet been defined, shows you a summary of the changes that will be made, and then displays real-time updates as your stack is being deployed. Once finished, you get a summary of the outcome along with any error messages collected along the way - including errors messages for stacks that have been rolled back and no longer exist.

https://github.com/aws-cloudformation/rain/blob/main/docs/README.tmpl#L24

May I add the sentence to README.tmlp?

Reply:
Actually I removed that one and haven't regenerated the README yet.